### PR TITLE
Update for issue #1722

### PR DESCRIPTION
**Pull request by KeilChris**
Documentation clean-up and ensuring packchk tool documentation is visible for the end user.

### CODE REVIEW

File: tools/README.md [hunk 1]
```diff
@@ -1,58 +1,20 @@
...
 
-**Package creation and validation**:
+## SVD file validation
 
-- [**packgen**](./packgen): create a software pack from a `CMake` based software repository.
-- [**packchk**](./packchk): semantic validation of a software pack description and the archive content.
+- [**svdconv**](https://open-cmsis-pack.github.io/svd-spec/main/svd_SVDConv_pg.html) validates CMSIS-SVD files and generates CMSIS-compliant device header files.
 
-**Package management** including discovery of components, devices, boards and examples:
+## Package creation and validation
```
Comment (jkrech) [inline tools/README.md:9]: I think it is worth mentioning the 
`gen_pack` https://github.com/Open-CMSIS-Pack/gen-pack/?tab=readme-ov-file#bash-library-for-gen-pack-scripts
here as well.

File: tools/projmgr/test/data/TestSolution/ComponentSources/components.Debug+RteTest_ARMCM3.cbuild.yml
Comment (jkrech) [inline tools/projmgr/test/data/TestSolution/ComponentSources/components.Debug+RteTest_ARMCM3.cbuild.yml:1]: This change seems unrelated to the documentation changes? Are you fixing a test case or is this a result from merging main?

File: tools/projmgr/docs/Manual/Proposals - OutofScope.md
Comment (jkrech) [inline tools/projmgr/docs/Manual/Proposals - OutofScope.md:1]: Maybe we need to retain this as there is some proposals captured?

**Review by jkrech — APPROVED** [review]
Just check wether you want to consider my comment

**Comment by KeilChris** [comment]
I have incorporated your feedback.

**Review by jkrech — APPROVED** [review]
LGTM

**Review by jkrech — APPROVED** [review]
LGTM
